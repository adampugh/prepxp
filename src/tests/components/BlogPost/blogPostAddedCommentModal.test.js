import React from "react";
import { shallow } from "enzyme";
import BlogPostAddedCommentModal from "../../../components/BlogPost/blogPostAddedCommentModal";

describe("<blogPostAddedCommentModal />", () => {
    test("should match snapshot", () => {
        const wrapper = shallow(<BlogPostAddedCommentModal isOpen={true} closeModal={() => {}}/>);
        expect(wrapper).toMatchSnapshot();
    });
});