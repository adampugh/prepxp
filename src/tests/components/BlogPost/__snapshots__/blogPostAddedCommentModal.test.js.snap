// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<blogPostAddedCommentModal /> should match snapshot 1`] = `
<Modal
  appElement={null}
  ariaHideApp={true}
  bodyOpenClassName="ReactModal__Body--open"
  className="modal"
  closeTimeoutMS={200}
  isOpen={true}
  onRequestClose={[Function]}
  parentSelector={[Function]}
  portalClassName="ReactModalPortal"
  shouldCloseOnEsc={true}
  shouldCloseOnOverlayClick={true}
  shouldFocusAfterRender={true}
  shouldReturnFocusAfterClose={true}
  style={
    Object {
      "overlay": Object {
        "background": "rgba(0, 0, 0, 0.6)",
        "zIndex": 2000,
      },
    }
  }
>
  <div
    className="modal--delete"
  >
    <span
      onClick={[Function]}
    >
      <i
        className="fas fa-times"
      />
    </span>
    <h2>
      Added comment
    </h2>
    <p
      className="modal__icon"
    >
      <i
        className="fas fa-check"
      />
    </p>
  </div>
</Modal>
`;
